packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx

```diff
@@ -18,8 +18,15 @@ import {
 
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the header */
   header?: React.ReactNode;
```

Comment:
It's not entirely apparent from the name, but this prop "Sets the collapsible header content". View the code under https://garden.zendesk.com/components/chrome#navigation-panel for details.

Reply:
Updated in latest push.